### PR TITLE
Add doc for searchAlias metadata

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

**Review by ger-benjamin — APPROVED** [review]
Looks good to me. Thanks for the tests
@jwkaltz can you take a look please (you have a good eye for the doc) ?

File: doc/administrator/administrate.rst.tmpl [hunk 1]
```diff
@@ -59,6 +59,10 @@ Some metadata items are used by the ``layers`` views (for editing) on the server
 * ``lastUpdateUserColumn``: Define a column which will be automatically filled with the user of the last edit.
 * ``readonlyAttributes``: Set attributes as read only.
 
+Some metadata items are used by the ``theme2fts` script:
+
+* ``searchAlias``: Comma separated list of search alias (keywords) to be added to the ``tsearch.ts``
```
Comment (jwkaltz) [inline doc/administrator/administrate.rst.tmpl:64]: of search aliases

File: doc/integrator/fulltext_search.rst [hunk 2]
```diff
@@ -72,6 +72,11 @@ A script is available to fill the full-text search table, for more information t
 
    docker-compose exec geoportal theme2fts --help
 
+Note that some tree item's metadata are used by the ``theme2fts` script:
```
Comment (jwkaltz) [inline doc/integrator/fulltext_search.rst:75]: item's -> items'
(because it is plural)

File: doc/administrator/administrate.rst.tmpl (hunk 1 above)
Comment (jwkaltz) [inline doc/administrator/administrate.rst.tmpl:64]: Is it clear what this tsearch.ts is, or what it stands for?
(It's not clear to me. I guess it means that a user search will match this element if it contains one of the search aliases).

**Review by jwkaltz — APPROVED** [review]

File: doc/administrator/administrate.rst.tmpl (hunk 1 above)
Comment (arnaud-morvan) [reply doc/administrator/administrate.rst.tmpl]: Hope it is clearer with:
```
* ``searchAlias``: Comma separated list of search aliases (keywords) to be added to the ``tsearch.ts``
  field for the considered tree item (theme, layer group or layer),
  that is to say that considered tree item will match when searching for the search aliases.
```

**Review by sbrunner — APPROVED** [review]

**Pull request merged**